source/plugin/configuration/loaders.rst

```diff
@@ -163,7 +162,7 @@ config should be overriden only if absent.
 If you have an extra configuration class, you can use a much easier approach that also works if the only a part of your
 config is missing. See also the examples on the :doc:`serialization` page.
 
-Example: Merging Nodes from an Asset
+Updating Configuration Files from the Default Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
need to change underline here to match title length, that's why compile failed

Reply:
Ah crap. Thought I did